### PR TITLE
Feature enum test

### PR DESCRIPTION
**Pull request by errea**
DESCRIPTION
Add Rspec test to each of the enumerable methods

my_each
my_each_with_index
my_select
my_all
my_any
my_none
my_injesct

Fixes # (issue)
[x]We created a test using Rspec to pass the enumerable methods some conditions
[x] fix the rubocop offenses

### CODE REVIEW

File: spec/enumerable_spec.rb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    it 'checking my_each is working' do
+      expect(words_array.my_each { |word| word }).to eq(words_array)
+      expect(hash.my_each do |key, value|
+               key if value == 'kiranitor'
+             end).to eql(hash.each do |key, value|
+                           key if value == 'kiranitor'
+                         end)
+    end
+  end
+
```
Comment (elisha2kyakpo1) [inline spec/enumerable_spec.rb]: Your code is not formatted properly.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hey @errea and @John-Arboleda, Good to see you guys reach this far!✔👍🎉.
All the tests are good and they passing, but there are few things you should address before your project can be approved.

1. Remove the commented code.
2. Your README file has no testing information like how to test, what to install to test, etc.
3. The code is not formatted properly.
4. You can disable the rubocop error for "Block has too many lines. [157/30]".

Please fix the above issues and submit the project for another review. 
If you have a question, please reach me on slack ✔🤣😂🎉
Elisha.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :red_circle: 

### Hello team, good job implementing the required changes. Now your readme looks great :+1: 

### To improve:
- [x] You should have one `it` block for each expected behaviour. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/itblock.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))

_**If you have any question related to the required changes feel free to comment here, and ping me on slack @GzimAsani**_

HAPPY CODING :computer: :desktop_computer:

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️
## Hi @errea and @kiranitor123  👩‍🚀 

Good job on implementing the changes as requested in the previous review. Well done.
**You can now merge and submit.**

    
**Keep being awesome devs and good luck with your next project!**
---------

Cheers and Happy coding!👏👏👏

Code review by **Dipesh Kumar**

(Feel free to contact me on slack by the handle of  **deep** if you have any additional question).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk)
[Twitter](https://twitter.com/97deepeshkumar)

**Pull request merged**